docs/Installation.md

```diff
@@ -148,15 +148,43 @@ Now to know how to update the `src/config/supported_distros.py` to reflect the n
 Step 2 of
 [Adding_new_distros](https://github.com/openmainframeproject/software-discovery-tool/blob/master/docs/Adding_new_distros.md#step-2-update-the-supported_distros-variable-in-configuration-file-sdt_basesrcconfigconfigpy)
 
-###  Step 6: Verify that the software-discovery-tool server is up and running
-* For Ubuntu (18.04, 19.04, 20.04, 22.04):
+### Step 6: Install and populate the SQL database
+* For Ubuntu (18.04, 20.04, 22.04):
+
+#### Install dependencies
+
+        sudo apt install mariadb-server python3-pymysql
+
+#### Run MariaDB from the command line and complete the secure installation. Remember the root password you set, you will need this in the future.
+
+        sudo mariadb
+        sudo mysql_secure_installation
```

Comment:
probably can move `mysql_secure_installation` up with the `mariadb-server` installation instruction

Reply:
Agreed. I actually am not sure why I have running mariadb in the command before here at all, since it's not necessary.